### ISSUE TITLE
SNES: update Gradius III / Race Drivin' SA-1 hacks to versions 1.6 / 1.1

### ISSUE THREAD

**Issue by tjanas** [frederic-mahe/Hardware-Target-Game-Database#627]
https://github.com/VitorVilela7/SA1-Root

https://twitter.com/HackerVilela/status/1378874110063828993?s=20

"both includes fixes and introduces addons for making the game speed stays close as original without lowering frame rate"

**Comment by birdybro** [comment]
Just did all of them, and the Gradius III SA-1 hack entries we have are unchanged when applying these patches bizarrely enough.

```
573abebee9eff9a73f7b51af803346893f7bd8d5f8d82152d978abcd0c2bd2c4	C:/emu/patch/parse/Contra III - The Alien Wars SA-1 (Hack) v1.1 Vitor Vilela.sfc	cd45b39bd12a08351a63c12c63bc6376d0453dae	25e20db0bb4635e0716a49e586bda96b	9005d3a1
17b9368f84e0c1d5173b80c6b53bb2dd5d4b6a608f6260ca37649f8045f8197f	C:/emu/patch/parse/Gradius III SA-1 Japan (Hack) v1.6 Vitor Vilela.sfc	868016c4e5fbf7453d8fc302b47cc2688250269a	059bf02a0cf5c9516b04667b98489693	e86aa13a
c6e4da68bee3e2e56d82f7e2e037f9066d24d07a49123752d4977c962e8fc170	C:/emu/patch/parse/Gradius III SA-1 Origin Japan (Hack) v1.6 Vitor Vilela.sfc	148762f1f8930b3b12a01d7ff9bb6de63bc1b9f9	817e2110b76ec0f46d3627089bef3560	3a58a5ff
9fc69e7b15660e4ce1526a79dce780440fcef39edfcab5ac2a97ae1999bef557	C:/emu/patch/parse/Gradius III SA-1 Origin USA (Hack) v1.6 Vitor Vilela.sfc	48548fac3857a042012d487f1483f7a33fae0ac8	1ba3c5181686f9ee0beade9a2637d38a	45611e5e
334119d66ae790d420c90a77dbc2ec2921e8a78b8a3f1222bda6acdb4922beeb	C:/emu/patch/parse/Gradius III SA-1 USA (Hack) v1.6 Vitor Vilela.sfc	c2a10546496a1c0ce7c2c1a8a747aee7af15a66a	6b79ec4484338b8a5715100fafda9a08	d8ec2c36
6ca9486bdf492228dcc75ba09a4364a6000ef96b0b63fb960f5f963fc906686b	C:/emu/patch/parse/Race Drivin' SA-1 (Hack) v1.1 Vitor Vilela.sfc	d69962546d29d5378992e29f56254e36086af9fe	c8ca265408b6536f0207a088b240a205	23eaa97e
6890dc6d69d656775ef70a5a353ca3639da119becdbb488b05b1ff31a37e01d7	C:/emu/patch/parse/Race Drivin' SA-1 Origin (Hack) v1.1 Vitor Vilela.sfc	4f54d94ca151bb2fc8d6ab9390baa9574c837abf	a75180a015cb193cbe1ac5bc01dd6486	bb42a085
0031c9a58353dc04e82541d7f93a42c925f6632289d727f30d6b727d67be2c1f	C:/emu/patch/parse/Super Mario World SA-1 (Hack) v1.40 Vitor Vilela.sfc	809af0309deec62786ec44c8203bf4394014c3ec	c0e7998863a2276a2f70722924e4bd38	389df7f4
9f777f1284eb4cc8a7707b99f7473aff80e50b33d0ee306b2193eb4aa5fc4cba	C:/emu/patch/parse/Super R-Type SA-1 (Hack) v1.1 Vitor Vilela.sfc	3fa6573ebb3a670ec48cdbcb2998f849f6c845b2	caa4ee0c6be38e505ec6100b36941e78	b50c346f
```

I'll submit a PR soon once I clean it all up. I'm adding the origin hack variants that were also missing prior.

**Comment by tjanas** [comment]
Interesting, the author of the patch indicated that the gameplay would change with the update to Gradius III:
https://twitter.com/HackerVilela/status/1378874113796755460?s=20

**Comment by birdybro** [comment]
I think it's possible their new patch isn't actually the new patch... I'll compare with 1.5 just to be sure, and raise an issue on their github to see if that's the problem.

**Comment by birdybro** [comment]
```
573abebee9eff9a73f7b51af803346893f7bd8d5f8d82152d978abcd0c2bd2c4	C:/emu/patch/patched-test/Contra III - The Alien Wars SA-1 (Hack) v1.1 Vitor Vilela.sfc	cd45b39bd12a08351a63c12c63bc6376d0453dae	25e20db0bb4635e0716a49e586bda96b	9005d3a1
17b9368f84e0c1d5173b80c6b53bb2dd5d4b6a608f6260ca37649f8045f8197f	C:/emu/patch/patched-test/Gradius III SA-1 Japan (Hack) v1.5 Vitor Vilela.sfc	868016c4e5fbf7453d8fc302b47cc2688250269a	059bf02a0cf5c9516b04667b98489693	e86aa13a
17b9368f84e0c1d5173b80c6b53bb2dd5d4b6a608f6260ca37649f8045f8197f	C:/emu/patch/patched-test/Gradius III SA-1 Japan (Hack) v1.6 Vitor Vilela.sfc	868016c4e5fbf7453d8fc302b47cc2688250269a	059bf02a0cf5c9516b04667b98489693	e86aa13a
c6e4da68bee3e2e56d82f7e2e037f9066d24d07a49123752d4977c962e8fc170	C:/emu/patch/patched-test/Gradius III SA-1 Origin Japan (Hack) v1.6 Vitor Vilela.sfc	148762f1f8930b3b12a01d7ff9bb6de63bc1b9f9	817e2110b76ec0f46d3627089bef3560	3a58a5ff
9fc69e7b15660e4ce1526a79dce780440fcef39edfcab5ac2a97ae1999bef557	C:/emu/patch/patched-test/Gradius III SA-1 Origin USA (Hack) v1.6 Vitor Vilela.sfc	48548fac3857a042012d487f1483f7a33fae0ac8	1ba3c5181686f9ee0beade9a2637d38a	45611e5e
334119d66ae790d420c90a77dbc2ec2921e8a78b8a3f1222bda6acdb4922beeb	C:/emu/patch/patched-test/Gradius III SA-1 USA (Hack) v1.5 Vitor Vilela.sfc	c2a10546496a1c0ce7c2c1a8a747aee7af15a66a	6b79ec4484338b8a5715100fafda9a08	d8ec2c36
334119d66ae790d420c90a77dbc2ec2921e8a78b8a3f1222bda6acdb4922beeb	C:/emu/patch/patched-test/Gradius III SA-1 USA (Hack) v1.6 Vitor Vilela.sfc	c2a10546496a1c0ce7c2c1a8a747aee7af15a66a	6b79ec4484338b8a5715100fafda9a08	d8ec2c36
6ca9486bdf492228dcc75ba09a4364a6000ef96b0b63fb960f5f963fc906686b	C:/emu/patch/patched-test/Race Drivin' SA-1 (Hack) v1.1 Vitor Vilela.sfc	d69962546d29d5378992e29f56254e36086af9fe	c8ca265408b6536f0207a088b240a205	23eaa97e
6890dc6d69d656775ef70a5a353ca3639da119becdbb488b05b1ff31a37e01d7	C:/emu/patch/patched-test/Race Drivin' SA-1 Origin (Hack) v1.1 Vitor Vilela.sfc	4f54d94ca151bb2fc8d6ab9390baa9574c837abf	a75180a015cb193cbe1ac5bc01dd6486	bb42a085
0031c9a58353dc04e82541d7f93a42c925f6632289d727f30d6b727d67be2c1f	C:/emu/patch/patched-test/Super Mario World SA-1 (Hack) v1.40 Vitor Vilela.sfc	809af0309deec62786ec44c8203bf4394014c3ec	c0e7998863a2276a2f70722924e4bd38	389df7f4
9f777f1284eb4cc8a7707b99f7473aff80e50b33d0ee306b2193eb4aa5fc4cba	C:/emu/patch/patched-test/Super R-Type SA-1 (Hack) v1.1 Vitor Vilela.sfc	3fa6573ebb3a670ec48cdbcb2998f849f6c845b2	caa4ee0c6be38e505ec6100b36941e78	b50c346f
```

Yup, same patch result, weird.

**Comment by birdybro** [comment]
False alarm, this is expected behavior.

The "Origin" version is what is new with 1.6. So this is correct!
